docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -48,3 +48,58 @@ If your issue has not been resolved by this procedure, perform the following ste
 . link:https://github.com/code-ready/crc/issues[Search open issues] for the issue that you are encountering.
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
+
+= Differences with a production OpenShift install
+
+{rh-prod} is a regular OpenShift install tweaked to be fit in a single VM on a laptop.
+While it tries hard to be as close as possible to a production deployment,
+there are some differences worth noting:
+
+* It uses a single node which behaves both as a master and as a worker node.
+* The `machine-config`, `marketplace` and `monitoring` operators are disabled by default.
```

Comment:
"Operators" should be capitalized according to the OpenShift documentation guidelines. :)